src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -0,0 +1,68 @@
+package analytics
+
+import (
+	"net/http"
+
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/getlantern/analytics"
+	"github.com/getlantern/flashlight/ui"
+	"github.com/getlantern/golog"
+)
+
+const (
+	messageType = `Analytics`
+)
+
+var (
+	log        = golog.LoggerFor("flashlight.analytics")
+	service    *ui.Service
+	httpClient *http.Client
+)
+
+func Configure(autoReport *bool, newClient *http.Client) {
+	httpClient = newClient
+
+	if autoReport != nil && *autoReport {
+		err := Start()
+		if err != nil {
+			log.Errorf("Error starting analytics service: %q", err)
+		}
+	}
```

Comment:
Actually the requirement is just that we don't send the data to the cloud.  You could keep the service running and just discard whatever data the UI sends it.